src/GraphQL/Types/Scalars/ULongGraphType.cs

```diff
@@ -2,16 +2,22 @@
 
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// The ULong scalar graph type represents an unsigned 64-bit integer value.
+    /// </summary>
     public class ULongGraphType : ScalarGraphType
     {
+        /// <inheritdoc/>
         public override object ParseLiteral(IValue value) => value switch
         {
             ULongValue ulongValue => ulongValue.Value,
             IntValue intValue => intValue.Value >= 0 ? (ulong?)intValue.Value : null,
             LongValue longValue => longValue.Value >= 0 ? (ulong?)longValue.Value : null,
+            BigIntValue bigIntValue => bigIntValue.Value >= 0 && bigIntValue.Value <= ulong.MaxValue ? (ulong?)bigIntValue.Value : null,
```

Comment:
I just want to ask one more time and make sure we are on the right track by adding such branches of code to many scalar classes?

Reply:
It will work just fine either way.  If we use `checked((ulong)bigIntValue.Value)` then overflows will throw an exception, get caught, and the proper exception is then thrown.  If we return null, it will detect that and throw the same type of exception.  I'm indifferent.

Reply:
I understand  but sooner or later someone will have a question why in some cases we use one, and in some other.

Reply:
Consistency is good.  In this case I copied the form used in the prior two lines of code.  We can always make another PR to consolidate all of the scalars to use the form in the ValueConverter if desired.

Reply:
ok